pyproject.toml

```
@@ -32,9 +32,8 @@ python = "^3.9"
 requests = "^2.31"
 iso8601 = "^2.1.0"
 orjson = {version = ">=3.9, <=3.10.3", optional = true} # 3.10.4 errors on install
-numpy = {version = "^1.26.3", optional = true}
-pandas = {version = "^2.1.4", optional = true}
-
+numpy = {version = ">=1.24.0", optional = true}
+pandas = {version = ">=2.0.0, <=3.0.0", optional = true}
```

Supporting 3.0.0 but nothing later doesn't seem right

```suggestion
pandas = {version = ">=2.0.0, <3.0.0", optional = true}
```